### PR TITLE
Issue2355 obc setpoint

### LINKED ISSUE #2355: Move CDL.SetPoints
This package has only one block which can be implemented using CDL.
This issue is therefore to move this package to `OBC.Utilities.SetPoints`.

As CLD has no function for `z = x^y`, this function will be implemented using `log` and `exp`.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #2355.

It also adds a validation case for m &ne; 1.

### CODE REVIEW

**Comment by mwetter** [comment]
@JayHuLBL : Can you please review this. Thanks.

File: Buildings/Controls/OBC/Utilities/SetPoints/SupplyReturnTemperatureReset.mo [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+  CDL.Interfaces.RealOutput TRet(
+    final quantity="ThermodynamicTemperature",
+    final unit = "K",
+    displayUnit = "degC",
+    min=200) "Setpoint for return temperature"
+    annotation (Placement(transformation(extent={{100,-80},{140,-40}})));
+
+protected
+  CDL.Continuous.Sources.Constant dTOutHeaBal_nominal(final k=dTOutHeaBal, y(
+        final quantity="TemperatureDifference", final unit="K"))
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/Utilities/SetPoints/SupplyReturnTemperatureReset.mo:59]: reformat the lines as below instances.

**Review by JayHuLBL — APPROVED** [review]
@mwetter It's ready to merge when the inline comment is addressed.

File: Buildings/Controls/OBC/Utilities/SetPoints/SupplyReturnTemperatureReset.mo (hunk 1 above)
Comment (mwetter) [reply Buildings/Controls/OBC/Utilities/SetPoints/SupplyReturnTemperatureReset.mo]: This has been updated now.

**Pull request merged**